R/Exclusion_criteria.R

```diff
@@ -0,0 +1,60 @@
+####exclusion criteria####
+#1. discontinued parkruns#
+#reorder
+df_event <- readRDS(file = "data/clean/dt_events_ts.rds")
```

Comment:
This file doesn't exist for me. This is because it is ignored by Git. To not ignore this file you need:

```
# Ignore everything
*

# But not these files...
!.gitignore
!script.pl
!template.latex
# etc...

# ...even if they are in subdirectories
!*/

# if the files to be tracked are in subdirectories
!*/a/b/file1.txt
!*/a/b/c/*

```